extend.ts

```diff
@@ -0,0 +1,9 @@
+class Base {
+  public base_prop = "base prop";
+}
```

Comment:
same as above, test file you committed?

Reply:
dooonnnne: [92bd978](https://github.com/drashland/rhum/pull/150/commits/92bd978cb4dff8f81caf474588f116654aab3f8a)